<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyCrypt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>RSA</string>
     </attribute>
     <widget class="QLabel" name="MainName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PyCrypt - RSA шифровальщик</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите два простых числа:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="anum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="bnum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushAB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Забить числа</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateKeys">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сгенерировать ключи</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="copyPub">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Копировать публичный</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copyPriv">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Копировать приватный</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="fileSave">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сохранять ключи в файлы</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputToX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Зашифровать:</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputToXkey">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите ключ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getToXkey">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="msgInpEnc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите сообщение:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="msgEnc">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="copyTextEnc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Копировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushKeyToX">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputForX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Расшифровать:</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputForXkey">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите ключ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getForXkey">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="msgInpDec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите сообщение:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="msgDec">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>400</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="copyTextDec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Копировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushKeyForX">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>380</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QLabel" name="warning">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>200</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Подборка простых числел</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>220</y>
        <width>201</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>tablicca_prostyz_chisel_do_1000.jpg</pixmap>
      </property>
     </widget>
     <widget class="QCheckBox" name="genNums">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сгенерировать числа</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="publicKey">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ваш публичный ключ: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="privateKey">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ваш приватный ключ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getForX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ваш расшифрованный текст будет тут</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="getToX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ваш зашифрованный текст будет тут</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="A_M">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ave Maria</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="PyCheck">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Python</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Помощь</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>771</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Инструкция
Предназначается для всяких Бондарей и ему подобных. 
Для начала введите два простых числа в поле ввода для чисел. 
Вы можете нажать на галочку &quot;сгенерировать числа&quot; и нажать &quot;забить числа&quot;.,
числа сгенерируются самостоятельно 
Далее нажмите &quot;сгенерировать ключи&quot;. 
Для зашифровки используйте публичный ключ. 
Для расшифровки используйте приватный ключ.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
